CONTRIBUTING.md

```diff
@@ -73,26 +73,14 @@ If you are making changes that do not need a version bump in any part of the rep
 
 ## Publishing via npm
 
-To publish a package to **npm**, please first ensure that you have registered an account on [npmjs.com](https://www.npmjs.com/).
-
-Internal contributors (JET Employees) should request to be added to the Just Eat npm organisation by asking in the #guild-frontend Slack channel and providing their npm username. This will then allow you to publish the packages from our mono-repo.
-
-To be able to publish you will need to run `npm login` first. You will also need to add your github access token to `.npmrc` file for components to be published to our Storybook. More info about that can be found [here](https://docs.github.com/en/packages/guides/configuring-npm-for-use-with-github-packages).
-
-External contributors should raise a pull request and the release will be handled for you once approved.
-
-Once you are happy that the package is ready to be published, ensure you are on the correct branch — typically the `master`/`main` branch with the latest changes pulled. Then run `npm publish` on the command line.
+Publishing to **npm** is handled automatically as part of our CI process. Once a PR has been merged to master, any package versions that are not present on npm will be published as part of the **master** build.
```

Comment:
Is it worth adding something about adding a tag manually once they are published (happy for this to be a separate PR/commit so you don't need to run the build again).